apps/desktop/package.json

```diff
@@ -92,6 +92,6 @@
     "proxy-agent": "^6.3.1",
     "react": "^18",
     "react-dom": "^18",
-    "styled-components": "^6.1.0"
+    "styled-components": "5.3.7"
```

Comment:
@devin-ai-integration you have downgrade. Can you try what happends if you upgrade to de last version 6.1.11